### PR TITLE
[Fix] Added Community Kicked notification to response

### PR DESCRIPTION
**Pull request by smohamedjavid**
Needed for https://github.com/status-im/status-mobile/pull/15734#issuecomment-1521929578

### What changed

This PR adds the Activity Center `Community Kicked` notification type to the response. So, the user's Activity Center gets updated instantly without user interaction.

### CODE REVIEW

**Review by ilmotta — APPROVED** [review]
An accompanying unit test would be good @smohamedjavid, but I have no further comments.

File: protocol/messenger.go [hunk 1]
```diff
@@ -4349,11 +4349,17 @@ func (m *Messenger) handleRetrievedMessages(chatWithMessages map[transport.Filte
...
 					}
 
 					err = m.addActivityCenterNotification(response, notification)
 					if err != nil {
-						m.logger.Error("failed to save notification", zap.Error(err))
+						logger.Error("failed to save notification", zap.Error(err))
+						continue
+					}
+
+					if err := messageState.Response.Merge(response); err != nil {
```
Comment (MishkaRogachev) [inline protocol/messenger.go:4359]: Ah, i missed here we are using `messageState.Response`. Thanks for fixing!

**Review by MishkaRogachev — APPROVED** [review]

**Comment by smohamedjavid** [comment]
> An accompanying unit test would be good @smohamedjavid, but I have no further comments.

Added 👍

File: protocol/messenger.go (hunk 1 above)
Comment (smohamedjavid) [reply protocol/messenger.go:4359]: No worries 😄

**Pull request merged**